ms2query/utils.py

```diff
@@ -105,6 +103,44 @@ def get_classifier_from_csv_file(classifier_file_name: str,
     else:
         results = pd.concat(list_of_classifiers, axis=0, ignore_index=True)
 
-    results["inchi_key"] = list_of_inchikeys
-    results.rename(columns={"inchi_key": "inchikey"}, inplace=True)
+    results["inchikey"] = list_of_inchikeys
     return results
+
+
+def columns_and_order_for_output_dataframe(return_standard_columns: bool,
+                                           return_classifier_columns: bool,
+                                           additional_metadata_columns: List[str] = None,
+                                           additional_ms2query_score_columns: List[str] = None) -> List[str]:
+    """Returns the column names in order for the output of results table
```

Comment:
It here says "column names in order" and in the function name also states "ordered", but I see no sorting or any other ordering step in the function.

Reply:
The ordered was used in the name here, since this output is also used for sorting the columns in the dataframe that is created.

Reply:
I changed it to make this less confusing